modules/administration-guide/pages/installing-image-puller-on-kubernetes-using-the-image-puller-operator.adoc

```diff
@@ -0,0 +1,6 @@
+[id="installing-image-puller-on-kubernetes-using-the-image-puller-operator"]
+:navtitle: Installing using the {image-puller-operator-name}
```

Comment:
@themr0c how will it be resolved for the CRW?

Reply:
Downstream is ignoring the content of the pages directory. It hops directly to the file referenced in the include statement.